src/libcmd/command.cc

```diff
@@ -14,12 +14,10 @@
 
 namespace nix {
 
-RegisterCommand::Commands * RegisterCommand::commands = nullptr;
-
 nix::Commands RegisterCommand::getCommandsFor(const std::vector<std::string> & prefix)
 {
     nix::Commands res;
-    for (auto & [name, command] : *RegisterCommand::commands)
+    for (auto & [name, command] : RegisterCommand::commands())
```

Comment:
I think we need to call these functions now in our manual.

Reply:
I think all we need to do is to move move those plugins into initNix() except for the Register::commands one.

Reply:
This was solved.